recipes/visit_struct/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "visit_struct" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/garbageslam/visit_struct/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7e5d727bd9fbe444d182840625f5eb21f2cf67f06f3aeab27a2d7c4724b3c09c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+
+test:
+  commands:
```

```suggestion
test:
  requires:
    - {{ compiler('cxx') }}
  source_files:
    - test_visit_struct.cpp
  commands:
    - $CXX test_visit_struct.cpp -o compile_test -I$PREFIX/include
    - ./compile_test
```
Can you please compile and run this file as a test. This will check that this header-only library is still compatible with our compilers. (compilers mentioned in the source are a few major versions old) Something similar for Windows too please.